apps/core/src/components/affine/setting-modal/account-setting/index.tsx

```diff
@@ -118,32 +118,32 @@ export const AccountSetting = () => {
         data-testid="account-title"
       />
       <AvatarAndName />
-      <SettingRow
-        name={t['com.affine.settings.email']()}
-        desc={user.email}
-        disabled={true}
-      >
-        <Button
-          onClick={() => {
-            toast('Function coming soon');
-          }}
-        >
-          {t['com.affine.settings.email.action']()}
-        </Button>
-      </SettingRow>
-      <SettingRow
-        name={t['com.affine.settings.password']()}
-        desc={t['com.affine.settings.password.message']()}
-        disabled={true}
-      >
-        <Button
-          onClick={() => {
-            toast('Function coming soon');
-          }}
-        >
-          {t['com.affine.settings.password.action']()}
-        </Button>
-      </SettingRow>
```

Comment:
You comment all of this? Sure?

Reply:
These function is not support in this version，and disable style not work well， They will come back with the new sign in function